src/utils/aws-kms-signer/create-gasless-account.ts

```
@@ -0,0 +1,21 @@
+import { Hex, LocalAccount } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { AwsKmsViemAccountConfig, kmsToAccount } from './viem-kms-account';
+
+export type GaslessAccountConfig =
+  | { type: 'privateKey'; privateKey: Hex }
+  | ({ type: 'kms' } & AwsKmsViemAccountConfig);
```

ESLint

[error] Replace `|·{·type:·'privateKey';·privateKey:·Hex·}⏎·` with `{·type:·'privateKey';·privateKey:·Hex·}`

(prettier/prettier)